data/init.sql

```diff
@@ -43,6 +43,26 @@ CREATE TABLE IF NOT EXISTS profiles(
       tags text[],
       CONSTRAINT name UNIQUE(profile_name)
     );
+CREATE TABLE IF NOT EXISTS wirelessconfigs(
+      wireless_profile_name citext NOT NULL,
+      authentication_method integer,
+      encryption_method integer,
+      ssid varchar(32),
+      pskvalue integer,
+      pskpassphrase varchar(63),
+      linkPolicy int[],
+      creation_date timestamp,
+      created_by varchar(40),
+      CONSTRAINT wirelessprofilename UNIQUE(wireless_profile_name)
+    );
+CREATE TABLE IF NOT EXISTS profile_wireless_configs(
```

Comment:
my goal with the name here is to match the referenced table names so should be `profiles_wirelessconfigs`

Reply:
done